yapapi/executor/__init__.py

```diff
@@ -212,25 +217,32 @@ async def create_demand_builder(
 
     @property
     def driver(self) -> str:
+        """Return the name of the payment driver used by this engine."""
         return self._driver
 
     @property
     def network(self) -> str:
+        """Return the name of the payment network used by this engine."""
         return self._network
 
     @property
     def storage_manager(self):
+        """Return the storage manager used by this engine."""
         return self._storage_manager
 
     @property
     def strategy(self) -> MarketStrategy:
+        """Return the instance of `MarketStrategy` used by this engine."""
         return self._strategy
 
-    def emit(self, *args, **kwargs) -> None:
+    def emit(self, event: events.Event) -> None:
+        """Emit an event to be consumed by this engine's event consumer."""
         if self._wrapped_consumer:
-            self._wrapped_consumer.async_call(*args, **kwargs)
```

Comment:
Were the args/kwargs here used anywhere?

Reply:
no, the only argument is always an `Event`